index.js

```diff
@@ -4,34 +4,242 @@ const closebtn = document.querySelector('.close-btn');
+  and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and
+  typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+  when an unknown printer took a galley of type and scrambled it 1960s with the relea`,
+    tecnologies: {
+      tech1: 'Rubi on rails',
+      tech2: 'css',
+      tech3: 'JavaScript',
+    },
+    linkVersion: 'https://www.google.com',
+    linkSource: 'https://adarachel.github.io/personal-portfolio/',
+  },
+  {
+    id: 5,
+    url: './img/newnew.svg',
+    title: 'Keeping track of hundreds of components',
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+  industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type
+  and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and
+  typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+  when an unknown printer took a galley of type and scrambled it 1960s with the relea`,
+    tecnologies: {
+      tech1: 'Rubi on rails',
+      tech2: 'css',
+      tech3: 'JavaScript',
+    },
+    linkVersion: 'https://www.google.com/',
+    linkSource: 'https://adarachel.github.io/personal-portfolio/',
+  },
+];
+
+const worksCardDetailsMobile = (index) => `
+  <article class="works-card-details-mobile">
+          <button type="button" class="close-btn-details">
+              <img src="./img/Icon.svg" alt="cancel">
+          </button>
+          <div class="works-card-image-details-mobile img">
+              <img src=${dataCards[index].url} alt="">
+          </div>
+          <h2 class="works-card-title-mobile">
+              ${dataCards[index].title}
+          </h2>
+          <ul class="works-card-skills-mobile">
+              <li class="works-card-skills-mobile li">${dataCards[index].tecnologies.tech1}</li>
+              <li class="works-card-skills-mobile li">${dataCards[index].tecnologies.tech2}</li>
+              <li class="works-card-skills-mobile li">${dataCards[index].tecnologies.tech3}</li>
+          </ul>
+          <p class="works-card-paragraph-mobile">
+            ${dataCards[index].description}
+          </p>
+          <div class="works-card-button-mobile">
+                  <a href=${dataCards[index].linkVersion} class="works-card-btn-details-mobile">
+                  <img src="./img/Enable.png" alt="see live">
+                  </a>
+                  <a href=${dataCards[index].linkSource} class="works-card-btn-details-mobile">
+                  <img src="./img/Enable.svg" alt="see github">
+                  </a>
+          </div>
+          <script src="index.js"></script>
+  </article>
+  `;
+
+const projectsCards = [
+  {
+    id: '0',
+    name: 'Multi-Post Stories Gain+Glory',
+    cardSkills: ['ruby on rails', 'css', 'Javascript', 'html'],
+    imageUrl: './img/doughnuts.jpeg',
+  },
+  {
```

Comment:
Good job! but don't you think you can use the `dataCards` array for this purpose? and your data will be centralized so when the user clicks on the `see project` button the popup will show the selected project.

Reply:
As to this particular issue, i have tried endlessly to use just the dataCards array but I have not been successful. I have asked help from coding partners, morning session teammates, stand up teammates and i have not been successful in fixing this issue. I have been working on this project since Tuesday, and i would very much like to be approved so that i can move on and catch up on wednesday and thursday's projects. If you can suggest something that will work, please do. If not please approve me and let me move on. Thank you.